cmd/minikube/cmd/config/addons_list.go

```diff
@@ -117,7 +117,7 @@ var printAddonsList = func(cc *config.ClusterConfig) {
 		klog.Errorf("list profiles returned error: %v", err)
 	}
 	if len(v) > 1 {
-		out.Step(style.Tip, "To see addons list for other profiles use: `minikube addons -p name list`")
+		out.Step(style.Tip, "To see addons list for other profiles use: `minikube addons -p name list`", false)
```

Comment:
lets move the true to Second arugment, so it looks easier to read.

Reply:
I agree, looks better in code too